examples/README.md

```diff
@@ -23,3 +23,7 @@ Or a PubSub example:
 ```
 make pubsub-python
 ```
+
+# Looking for more function examples?
+
+You can find more examples at https://github.com/kubeless/functions
```

Comment:
make that a URL link so we can click

Reply:
it's actually clickable once it's rendered:

<img width="759" alt="screen shot 2018-01-08 at 15 10 34" src="https://user-images.githubusercontent.com/4025665/35516101-2ad4dae0-050b-11e8-9ca7-5076a0617a9b.png">

but yes, better specify it

Reply:
It's actually clickable once it is rendered:
<img width="936" alt="screen shot 2018-01-29 at 15 42 35" src="https://user-images.githubusercontent.com/4025665/35516175-5c7be624-050b-11e8-87ce-c746e2a8ac29.png">
but yes, better be explicit